.circleci/config.yml

```diff
@@ -66,6 +66,12 @@ jobs:
             - server/node_modules
           key: server-{{ checksum "package-lock.json" }}
 
+      # Authenticate GCloud
+      - run: echo $GCLOUD_SERVICE_KEY | base64 --decode --ignore-garbage > ${HOME}/gcloud-service-key.json
```

Comment:
Doing deployment in a script could improve readability 😄, but nothing mandatory.